plugins/prettier/src/tasks/prettier.ts

```diff
@@ -10,7 +10,7 @@ export default class Prettier extends Task<typeof PrettierSchema> {
   static description = ''
 
   static defaultOptions: PrettierOptions = {
-    files: ['**/*.js'],
+    files: ['**/*.{js,json,jsx,ts,tsx}'],
```

Comment:
**task:** Remove the `json` extension. This will catch `package.json` and `package-lock.json` which could get quite messy without a default ignore pattern for at least the lockfile.

Reply:
Although it looks like prettier formats `package.json` and `package-lock.json` the same way `npm` does now: https://github.com/prettier/prettier/blob/24d39a906834cf449304dc684b280a5ca9a0a6d7/website/blog/2018-05-27-1.13.0.md#added-json-stringify-parser-for-jsonstringify-style-formatting-4450-by-ikatyang